src/content.js

```diff
@@ -1,4 +1,31 @@
 // this is the code that "Runs" the extension.
 // This is seperate from the code that runs the UI inside the extension.
-
 const deliberatelybroke = true;
+
+const videoTags = $('meta[name=\'keywords\']').attr('content').toLowerCase();
+const videoTitle = $('meta[name=\'title\']').attr('content').toLowerCase();
+const videoDesc = $('meta[name=\'description\']').attr('content').toLowerCase();
+let isMusic;
+
```

Comment:
I don't really understand what the for loop does. It looks like you're performing three disparate tasks; the switch statement effectively cancels the efficacy of the loop entirely.